dev/stablesats.yml

```diff
@@ -0,0 +1,19 @@
+pubsub:
+  host: redis
+
+price_server:
+  enabled: true
+  server:
+    listen_port: 3325
+  fees:
+    base_fee_rate: 0.005
```

Comment:
this represents 50 bps? I think the dealer v1 had more like 5 bps as the default value

Reply:
You're right there is a `0` missing https://github.com/GaloyMoney/dealer/blob/e22cad5c61909cf532c3f4eb549fcb0673f3d176/dealer/test/integration/DealerPriceService.spec.ts#L16-L18